WcaOnRails/app/helpers/static_pages_helper.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 module StaticPagesHelper
-  def format_team_members(team_friendly_id)
-    Team.find_by_friendly_id!(team_friendly_id).current_members.includes(:user).order(team_leader: :desc).order("users.name asc").map do |u|
-      u.user.name + (u.team_leader ? " (leader)" : "")
+  def format_team_members(slug)
+    Team.find_by_slug!(slug).current_members.includes(:user).includes(:committee_position).order("committee_positions.team_leader desc").order("users.name asc").map do |u|
```

Comment:
You can pass multiple arguments to `includes`, like `includes(:user, :committee_position)` =)